docs/contributor-guide/how-to-write-docs.md

```diff
@@ -0,0 +1,32 @@
+# How to write docs
+
+When writing user-facing documentation, pick one of the four options:
+1. Tutorial
+2. How-To guide
+3. Explanation
+4. Reference
+
+For more, see descriptions below or [Divio's documentation system manual](https://documentation.divio.com/).
+
+### Tutorial
+
+Tutorials are learning-oriented and ...
+- must walk-through a self-contained domain like "Deduplicating buildings in Cambridge".
+- must provide any necessary configuration and data for the tutorial.
+
+### How-To guide
+
+How-To guides are problem-oriented and ...
+- must answer a specific, domain-agnostic question like "How to stream datasets out of Tamr"
```

Comment:
- Must include a list of numbered steps, known as a _task_, or a _procedure_, to help users complete a specific, domain-agnostic task, such as running a request, copying a file, installing, exporting, or other. For example, you can create a task titled "How to stream datasets out of Tamr".
- Must include a context paragraph, such as "It is often useful to stream datasets from Tamr, to load them into business analytics applications, such as Tableau, for analysis." Context may also include checks needed to be in place before users start the task, and links to related concepts. Context must provide information needed to begin the task, such as, it can list the host and port URL at which the endpoint for the service is served.
 - Must include a stem sentence, such as: "To stream a dataset out of Tamr:" The stem sentence is followed by numbered steps.
  - Must include a numbered list of steps where each step must begin with an imperative verb, such as: "Run the following `curl` request.", or "Save the file". For more examples see [Use Imperatives in Procedures](http://www.cs.cmu.edu/afs/cs.cmu.edu/project/cmt-40/kantoo/vol40/doc/kce/styleguide/imperatives.html).